orchagent/intfsorch.cpp

```diff
@@ -588,7 +588,7 @@ void IntfsOrch::doTask(Consumer &consumer)
         string vrf_name = "", vnet_name = "", nat_zone = "";
         MacAddress mac;
 
-        uint32_t mtu;
+        uint32_t mtu = 0;
```

Comment:
Port object is not supposed to be created with "INTF_TABLE:Ethernet4.20:192.169.1.1/24"

I believe on line 745, we need to amend the condition to create Port object as `&& !ip_prefix_in_key`

Reply:
Thanks. Done.